### PR TITLE
Fixes next/previous buttons on desktop

### PR DESCRIPTION
**Pull request by prateekbh**
Adds event listeners for next previous buttons on desktop and test for it. 

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

File: extensions/amp-story/0.1/amp-story.js [hunk 1]
```diff
@@ -329,6 +329,17 @@ export class AmpStory extends AMP.BaseElement {
   }
 
   /** @private */
+  addButtonListeners_() {
```
Comment (choumx) [inline extensions/amp-story/0.1/amp-story.js]: Let's remove this method and move the code into `buildButtons_()`. I don't think there's a case where we want to do one without the other.

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: I feel putting it in same function kinda makes unit testing a little tougher

Comment (choumx) [reply extensions/amp-story/0.1/amp-story.js]: You can test the behavior of `buildButtons_()` instead.

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: yep, but if that same function creates the button and also adds the listener there's nothing to stub as I am doing right now.

Comment (choumx) [reply extensions/amp-story/0.1/amp-story.js]:
```js
it('adds event listener for buttons', () => {
  story.buildButtons_();
  const nextStub = sandbox.stub(story, 'next_');
  this.nextButton_.dispatchEvent(new Event('click'));
  expect(nextStub).calledOnce;
});
```

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: Done

**Pull request merged**